### ISSUE TITLE
VM service: FlagList documentation is incorrect

### ISSUE THREAD

**Issue by nex3** [dart-lang/sdk#24145]
The documentation for the `FlagList` type is:

```
class FlagList extends Response {
  // A list of all flags which are set to default values.
  Flag[] unmodifiedFlags;

  // A list of all flags which have been modified by the user.
  Flag[] modifiedFlags;
}
```

However, the actual JSON response just contains a single `flags` key that combines the modified and unmodified flags.

**Comment by turnidge** [comment]
Yes.  Spec is wrong.  Will fix.

**Comment by turnidge** [comment]
Fixed in bfd59ed7993dc8718b2beb501b429a69fc4c1973.
